tests/test_repositories.py

```diff
@@ -170,6 +204,29 @@ def test_create_remote_repository_success(mocker):
     assert remote_repo == REMOTE_REPOSITORY_RESPONSE
 
 
+@responses.activate
+def test_create_federated_repository_success(mocker):
+    responses.add(responses.GET, f"{URL}/api/repositories/{FEDERATED_REPOSITORY.key}", status=404)
+    responses.add(
+        responses.PUT,
+        f"{URL}/api/repositories/{FEDERATED_REPOSITORY.key}",
+        json=FEDERATED_REPOSITORY_RESPONSE.model_dump(),
+        status=201,
+    )
+    responses.add(
+        responses.GET,
+        f"{URL}/api/repositories/{FEDERATED_REPOSITORY.key}",
+        json=FEDERATED_REPOSITORY_RESPONSE.model_dump(),

```

Comment:
I don't think those tests work since `FEDERATED_REPOSITORY_RESPONSE` is missing the `members` key

Reply:
aand again :D see above